### ISSUE TITLE
[FEATURE REQUEST] Need more detailed and accurate documentation

### ISSUE THREAD

**Issue by lanlin** [spiral/roadrunner#374]
hi there,

I find the documentation may out of date.

some configuration property not found in there, like `-c` `-w` `-l`

does all property has `enable`? or just remove it when not needed, like `static`

**Comment by 48d90782** [comment]
@lanlin Yes, there is no documentation from these flags, but you can use `rr help` to show description for all the CLI flags.
`-c` -> `config file (default is .rr.yaml)`
`-l` -> `select log formatter (color, json, plain)`
`w`  -> `working directory`

**Comment by 48d90782** [comment]
Thanks for the proposal, we will add all CLI flags to the docs.

**Comment by lanlin** [comment]
@48d90782 thanks a lot
